clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/chat/views/skins/NewMessagesIndicatorSkin.mxml

```diff
@@ -18,44 +17,31 @@
 				if (cornerRadius != cr) {
 					cornerRadius = cr;
 					background.radiusX = cornerRadius;
-					border.radiusX = cornerRadius;
 				}
 				
 				backgroundFill.color = getStyle("backgroundColor");
 				backgroundFill.alpha = getStyle("backgroundAlpha");
 				
-				borderStroke.color = getStyle("backgroundColor");
-				borderStroke.alpha = getStyle("backgroundAlpha");
-				
-				labelDisplay.setStyle("color", getStyle("color"));
-				labelDisplay.setStyle("fontSize", getStyle("fontSize"));

```

Comment:
All styles are now in Default.css `libChat|NewMessagesIndicator` and applied via `updateDisplayList` function override.